hw/formal/README.md

```diff
@@ -1,7 +1,4 @@
----
-title: "OpenTitan Assertions"
-unlisted: true
----
+# OpenTitan Assertions
 
 # OpenTitan Assertions
```

Comment:
This section heading has the same importance as the title.

Reply:
Could we fix this in a future PR with other content changes?